docs/en/Best-Practices/Index.md

```diff
@@ -24,4 +24,5 @@ Also, this guide is mostly usable for general **application development**.
 * Data Access
   * [Entity Framework Core Integration](Entity-Framework-Core-Integration.md)
   * [MongoDB Integration](MongoDB-Integration.md)
+  * [PostgreSQL Integration](PostgreSQL-Integration.md)
```

Comment:
PostgreSQL Integration should be a sub-item of Entity Framework Core Integration, what do you think?